barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_execution.hpp

```diff
@@ -17,6 +17,10 @@ class Execution {
     static std::vector<Row> gen_trace(std::vector<Instruction> const& instructions,
                                       std::vector<FF> const& calldata = {});
     static bb::HonkProof run_and_prove(std::vector<uint8_t> const& bytecode, std::vector<FF> const& calldata = {});
+
+    static std::tuple<AvmFlavor::VerificationKey, bb::HonkProof> prove_vk(std::vector<uint8_t> const& bytecode,
+                                                                          std::vector<FF> const& calldata = {});
+    static bool verify(AvmFlavor::VerificationKey vk, const HonkProof& proof);
```

Comment:
We usually have the style where "const" keyword is after the type: "HonkProof const& proof"